angular.json

```diff
@@ -0,0 +1,11 @@
+{
+"root": "",
+"sourceRoot": "src",
+"projectType": "application",
+"prefix": "app",
+"schematics": {},
+"architect": {
+  "styles": [
+  ]
+}
+}
```

Comment:
@strausr , what's that?

Reply:
Removed. It's for adding an external css to the components